doc/api/n-api.md

```diff
@@ -4471,6 +4472,11 @@ prevent the event loop from exiting. The APIs `napi_ref_threadsafe_function` and
 
 <!-- YAML
 added: v10.6.0
+napiVersion: 4
+changes:
+  - version: v10.17.0
```

Comment:
Based in discussion in other PR seems like this should also be version: REPLACEME here as well.